config/processEimage.py

```diff
@@ -0,0 +1 @@
+config.calibrate.photoCal.doSelectUnresolved=False
```

Comment:
As a practical matter, we need to be able to run the Twinkles workflow using conda distributions.  I don't have the time or resources to build from source.  I think someone else better do this review.